docker/run_jaxbtck.sh

```diff
@@ -43,8 +43,21 @@ if [ -z "$JAVA_HOME" ]; then
   export JAVA_HOME=/opt/jdk1.8.0_171
 fi
 
+if [[ "$JDK" == "JDK11" || "$JDK" == "jdk11" ]];then
+  export JAVA_HOME=${JDK11_HOME}
```

Comment:
Going forward, JDK 8 will become of the lesser priority, so I'd perhaps hide JDK 8 behind `if`s instead. It would be also nice to treat current JDK as JDK11 by default without any checks, so one can use JDK 11 or newer (ie JDK 15) to run tests locally and let the infra use what is required